specification/k3po.control/SPEC.md

```diff
@@ -50,7 +50,7 @@ are formatted as follows:
 
 If the "content-length" header is specified, then a message body of the specified length must be added.
 
-Headers and messages that are not understood/processed by a receiver may be ignored. The writers must only send headers and messages that it understands. 
+Headers and messages that are not understood/processed by a receiver should be ignored. The writers must only send headers and messages that it understands. 
```

Comment:
Please change this to the following for precision (in general we should be following RFC convention and using SHOULD, MUST, MUST NOT, etc rather than should, must, must not, etc for specification content)

Receivers SHOULD ignore undefined message headers and message types. Senders MUST NOT send undefined message headers or message types.